drivers/SmartThings/matter-lock/src/init.lua

```diff
@@ -476,7 +479,26 @@ local function handle_name_slot(driver, device, command)
   end
 end
 
-local function device_init(driver, device) device:subscribe() end
+local function find_default_endpoint(device, cluster)
+  local res = device.MATTER_DEFAULT_ENDPOINT
+  local eps = device:get_endpoints(cluster)
+  table.sort(eps)
+  for _, v in ipairs(eps) do
+    if v ~= 0 then --0 is the matter RootNode endpoint
+      return v
+    end
+  end
+  return res
```

Comment:
Thoughts on adding a warning message here? I think we don't expect to ever hit this so it'd be helpful to log if we are

Reply:
Added log message in this function for each driver 👍

Reply:
Looks like this one might have gotten missed

Reply:
welp, forgot to save the file 🤦, should be good now!